### PR TITLE
 python312Packages.plyara: init at 2.1.1

### PR DESCRIPTION
**Pull request by 13621**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/python-modules/plyara/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  setuptools,
+  wheel,
+  ply,
+}:
+
+buildPythonPackage rec {
+  pname = "plyara";
+  version = "2.1.1";
+  pyproject = true;
+
```
Comment (getchoo) [inline pkgs/development/python-modules/plyara/default.nix:14]:
```suggestion

  disabled = pythonOlder "3.10";

```
A nice to have for those still using older Python versions

File: pkgs/development/python-modules/plyara/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    ply
+  ];
+
+  pythonImportsCheck = [
+    "plyara"
+  ];
+
+  meta = {
+    description = "Parse YARA rules";
+    homepage = "https://pypi.org/project/plyara/";
```
Comment (getchoo) [inline pkgs/development/python-modules/plyara/default.nix:35]:
```suggestion
    homepage = "https://pypi.org/project/plyara/";
    changelog = "https://github.com/plyara/plyara/releases/tag/v${version}";
```

**Review by getchoo — CHANGES_REQUESTED** [review]
And welcome to Nixpkgs :)

**Comment by 13621** [comment]
@getchoo thank you :)

**Comment by getchoo** [comment]
@ofborg build python311Packages.plyara python312Packages.plyara

**Review by getchoo — APPROVED** [review]
Thanks for the work here. Feel free to ping me in a couple days if there are no other reviews

**Comment by 13621** [comment]
@getchoo hi, mind taking another look?

**Pull request merged**